src/handlers/events/delivery/goals/executeGoal.ts

```diff
@@ -50,7 +50,7 @@ export async function executeGoal(rules: { projectLoader: ProjectLoader },
     const implementationName = sdmGoal.fulfillment.name;
     logger.info(`Running ${sdmGoal.name}. Triggered by ${sdmGoal.state} status: ${sdmGoal.externalKey}: ${sdmGoal.description}`);
 
-    await markGoalInProcess(ctx, sdmGoal, goal);
+    await markGoalInProcess({ctx: ctx, sdmGoal: sdmGoal, goal: goal, progressLogUrl: progressLog.url});
```

Comment:
Use shorthand for `ctx` and `goal`

Reply:
do you mean, don't use shorthand?

Reply:
... oh you mean change it to say "ctx, goal"
This was a refactor, intellij does it this way